wit/main.wit

```diff
@@ -1 +1,9 @@
-package unused:agentic;
+package golem:agentic-guest;
+
+world agentic-guest {
+    import golem:api/host@1.1.7;
```

Comment:
You can include the world from `golem:agent` using `include golem:agent/agent-guest` to avoid duplicate definition